src/library_wasi.js

```diff
@@ -62,6 +62,33 @@ var WasiLibrary = {
     });
     return 0;
   },
+
+  args_sizes_get: function(argc, argv_buf_size) {
```

Comment:
please prefix with `p` the out params that are pointers, for consistency with the others (so, `pargc, pargv_buf_size`).

or if you don't like that let's discuss changing the convention.

Reply:
Sure.